content/topics/file-and-directory-naming/python/_index.md

```diff
@@ -12,39 +12,64 @@ title: File and directory naming in Python
 
 ## How should I name my Python modules and packages?
 
-You should name your files and directories the same way you name Python variables.They should be small letters, and words should be separated by underscores, as that improves readability.
+When naming files and directories in Python, you should use small letters and underscores to separate your words.
+This type of naming convention is called snake_case and is also used in naming Python variable and function names.
 
-It is important to note that the use of underscores in directory_naming takes preference as opposed to the use of dashes.
+### You think you got it? 
 
-Also, be sure to keep in mind that all Python files have a .py extension. See below the recommended Python file and directory naming style. 
+#### Directories
 
-### NO:
+Which of the following is a correct way to name a Python directory?
 ```
-TASK
-    1task.py
-    task 1.py
-    task1.1.py
-        
-passwordChecker
-    add-logger.py
-
-snakecase
-    productionnotes.py
-    keepItShortAndmeaningful.py
+  > apyhthondirectory
+  
+  > a_python_directory
+  
+  > a-python-directory
+  
+  > APYTHONDIRECTORY
+  
+  > ApythonDirectory
 ```
 
-### Yes:
+We are curious as to what your answer was!
+See below for some tips in naming your directories.
+```
+  > good_directory_name
+  
+  > bad-directory-name
+  
+  > AnotherBadDirectoryName
+  
+  > AVERYBADDIRECTORYNAME
+  
+  > nevernameyourdirectorylikethis
+```
+#### Files
 
+Is any of the examples below up to our recommended naming standard?
```

Comment:
Test yourself again - do any of the examples below follow the recommended Python naming guidelines?